backend/user/views.py

```diff
@@ -150,6 +156,70 @@ def delete(self, request, admin_id):
         return Response(status=status.HTTP_204_NO_CONTENT)
 
 
+class SaveUnitApiView(APIView):
+    """
+    An endpoint to handle saving a vehicle to a bidder's saved vehicle list,
+    as well as retrieving a list of all saved vehicles
+    """
```

Comment:
@kevinrczhang This endpoint would make more sense in the auctions app. Let's move the View to Auctions.